src/elements/content-preview/ContentPreview.js

```diff
@@ -784,6 +789,7 @@ class ContentPreview extends React.PureComponent<Props, State> {
             fileOptions: fileOpts,
             header: 'none',
             headerElement: `#${this.id} .bcpr-PreviewHeader`,
+            previewExperiences,
```

Comment:
Minor, but since we're passing this value to preview, should we go with just `experiences` here?

Reply:
yes, I like that idea